src/main/kotlin/racingcar/Application.kt

```diff
@@ -1,5 +1,97 @@
 package racingcar
 
+import camp.nextstep.edu.missionutils.Console
+import camp.nextstep.edu.missionutils.Randoms
+
 fun main() {
-    // TODO: 프로그램 구현
+    println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")
+    val inputCar = Console.readLine()
+    var cars = inputCar.split(",")
+    isValidCarName(cars)
+    cars = duplicateName(cars)
+
+    println("시도할 횟수는 몇 회인가요?")
+    val inputMove = Console.readLine()
+    val move = isMovementValid(inputMove)
+    race(cars, move)
+}
+
+fun isValidCarName(cars: List<String>){
+    isOver5Letters(cars)
+    isNameless(cars)
+}
+
+fun isOver5Letters(cars: List<String>) {
+    cars.forEach{ car ->
+        if (5 < car.length) {
+            throw IllegalArgumentException("이름이 5자를 넘습니다")
+        }
```

Comment:
```suggestion
        require(car.length <= 5) { "자동차 이름은 5자 이하여야 합니다." }
```
이번 2주차 과제를 진행하면서 코틀린의 require() 함수를 알게 되었습니다.
require()는 인자의 조건이 false일 때 중괄호 안의 메시지로 IllegalArgumentException을 발생시키며
코틀린에서 권장하는 방법이라고 하니 사용 해보시는걸 추천드립니다!

Reply:
와! 역시 코틀린! 자바보다 맛있다!
처음 알았어요 require(), 앞으로 쓰면서 애용하겠습니다!